mkdocs.yml

```diff
@@ -253,8 +253,8 @@ nav:
     - Tooltips: reference/tooltips.md
   - Insiders:
     - insiders/index.md
-    - Getting started:
-      - Installation: insiders/getting-started.md
-      - Changelog: insiders/changelog.md
+    - Getting started with Insiders: insiders/getting-started.md
```

Comment:
I'd drop "with Insiders" - we have it in the page title and in the menu it's redundant, because it's the first navigation item of the section that says "Insiders".